.gitignore

```diff
@@ -1,3 +1,8 @@
 /node_modules
 /image_recognition/data/all_images
-/pictures
\ No newline at end of file
+/pictures
+image_recognition/data/break/
+image_recognition/data/continue/
+.vscode/settings.json
+image_recognition/data/__pycache__/
+image_recognition/data/.ipynb_checkpoints/
```

Comment:
Would it help to exclude `image_recognition/data` as a whole?
And maybe we could have `.vscode/settings.json` in our repo?

Reply:
I see, there are Python files in `image_recognition/data`, but maybe they could be moved one folder up?